src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java

```diff
@@ -250,56 +267,161 @@ public class MongoSourceConfig extends AbstractConfig {
           + "watched.";
   private static final String COLLECTION_DEFAULT = EMPTY_STRING;
 
-  public static final String COPY_EXISTING_CONFIG = "copy.existing";
+  public static final String STARTUP_MODE_CONFIG = "startup.mode";
+  private static final String STARTUP_MODE_CONFIG_DISPLAY =
+      "The behavior when there is no offset data.";
+  private static final String STARTUP_MODE_CONFIG_DOC =
+      format(
+          "A change stream cannot be resumed if there is no offset data."
```

Comment:
The first sentence is a little confusing in its relationship to the config.

Suggest simplifying the initial description to something like "The initial connector start up mode."

Then later put a note stating that once the connector has been started and change stream offset data is stored any restarts will continue from that offset.

Reply:
What does "initial" means here and [below](https://github.com/mongodb/mongo-kafka/pull/125#discussion_r1018830236)? The only meaning I can think of based on your suggestion is "when there is no offset data". If that's the case, then what's the purpose of introducing a new term "initial start up" given that we don't use it anywhere else, including the documentation in question?

> Then later put a note stating that once the connector has been started and change stream offset data is stored any restarts will continue from that offset.

Note that `offset.partition.name` explains nothing about what "offset" means. This seems reasonable given that it's the Connect Framework term, not a connector-specific term.

Reply:
I updated the doc, please take a look.